docs/technical-documentation/alpaca_tips.md

```diff
@@ -0,0 +1,38 @@
+# Alpaca Tips
+
+[Alpaca](https://github.com/Islandora/Alpaca) is event-driven middleware based on [Apachel Camel](https://camel.apache.org/) for Islandora
+
+Currently, Alpaca ships with four event-driven components which are being run with [Apache Karaf](https://karaf.apache.org/)
+- islandora-connector-derivative
```

Comment:
Since you describe each of these in turn (well, at least make sections for each of them) you can link them:
```
- [islandora-connector-derivative](#islandora-connector-derivative)
```